jenkins/dockerfiles/dt-validation/Dockerfile

```diff
@@ -9,4 +9,10 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
 
 RUN update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-8 500
 
-RUN pip3 install git+https://github.com/devicetree-org/dt-schema.git@master
```

Comment:
Is it necessary to move this `install` command to the bottom?
It may result in unnecessary layer creation during image update from older version